### PR TITLE
WRO-784: Update React dependencies to version 18

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

Updated to React18 and to new React syntax

### CODE REVIEW

File: Icon/Icon.js [hunk 1]
```diff
@@ -50,7 +49,8 @@ const IconBase = kind({
 		 * @type {String|Object}
 		 * @public
 		 */
-		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
+		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+		css: PropTypes.object
```
Comment (daniel-stoian-lgp) [inline Icon/Icon.js:61]: can you add a short description for this prop?
Also, keep an empty line between props

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**